tests/ui/conftest.py

```diff
@@ -47,61 +56,38 @@ def selenium(selenium, request):
     Mobile size: 738x414 (iPhone 7+)
 
     """
+    # Skip mobile test with marker 'desktoponly'
+    if request.node.get_marker("desktoponly") and request.param == (414, 738):
+        pytest.skip("Skipping mobile test")
```

Comment:
That's kinda ugly. Let's move the resolution params into constants and re-use them when needed.

```python
DESKTOP = (1080, 1920)
MOBILE = (414, 738)

if request.node.get_marker('desktoponly') and request.param == MOBILE:
    # ...
```